bokehjs/examples/anscombe/anscombe.ts

```diff
@@ -1,8 +1,8 @@
 namespace Anscombe {
   console.log(`Bokeh ${Bokeh.version}`);
-  Bokeh.set_log_level("debug");
+  Bokeh.set_log_level("info");
 
-  const anscombe_quartet = Bokeh.LinAlg.transpose([
+  const anscombe_quartet = Bokeh.Utils.transpose([
```

Comment:
This will require a migration note, just I'm still not sure if this is the final API.